website/server/libs/cron.js

```diff
@@ -266,6 +293,7 @@ export function cron (options = {}) {
   // On cron, buffs are cleared and all dailies are reset without performing damage
   if (user.preferences.sleep === true) {
     performSleepTasks(user, tasksByType, now, daysMissed);
+    ageTodos(user, tasksByType, true, daysMissed, 0);
```

Comment:
Should this call to `ageTodos` be moved into `performSleepTasks` for consistency? Maybe also move `trackCronAnalytics` (from the line just below).

Although see also my point below about how performSleepTasks troubles me. :)

I think the `true` value in the call to `ageTodos` should be replaced by `multiDaysCountAsOneDay` in case we make that a user preference setting later.

Reply:
Eh, maybe not. I think `performSleepTasks` has too many responsibilities right now. All the arguments are off. I'm slowly making some refactoring as you can see which will allow for better argument passing.

Reply:
I have moved up the scope of `multiDaysCountAsOneDay` and passed it!